client/my-sites/site-settings/traffic/index.js

```diff
@@ -6,12 +6,9 @@ import page from 'page';
 /**
  * Internal dependencies
  */
-import config from 'config';
 import controller from './controller';
 import mySitesController from 'my-sites/controller';
 
 export default function() {
-	if ( config.isEnabled( 'manage/seo' ) ) {
```

Comment:
Why are we taking out this check? If `manage/seo` turned on in all environments? If it is, can we also remove those keys from the config files in this PR?